.github/workflows/publish.yaml

```diff
@@ -0,0 +1,35 @@
+name: Publish package
+
+on:
+  push:
+    # branches:
+    #   - main
```

Comment:
Does this need to be uncommented before merging?

Reply:
This build action requires testing before merging.